### ISSUE TITLE
Probably a request

### ISSUE THREAD

**Issue by MarioPaladinWeb** [DanielDrabik/Contact-Form-7-Range-Slider-Extender#1]
Thanks for the plugin. 

I hope it is possible to place a dollar sign and a comma whenever it reaches a thousand or a million value? Is this possible sir? Thanks!

**Comment by DanielDrabik** [comment]
To attach dollar sign simply add this to css file

`.contactform7-output:before {
  content: "$";
}
`

Second request is probably not possible. I've tried with js, but I couldn't override the output's inner html.
`jQuery('.contactform7-output').bind("DOMSubtreeModified",function(){  
var number = Number(jQuery(this).html());
    jQuery(this).html(number.toLocaleString('en-US', {style: 'currency', currency: "USD"}));
})`

My browser just crashed from number of errors I got.
